pandas/core/series.py

```diff
@@ -5635,6 +5783,7 @@ def dropna(self, axis=0, inplace=False, how=None):
                 pass
             else:
                 return self.copy()
+        return None
```

Comment:
Nit: Maybe just replace `pass` with `return None` above?

Reply:
I simplified the if statement but I still keep the `return` at the end because there is another if-else above that does not return.